src/heuristics.jl

```diff
@@ -19,3 +19,23 @@ function find_best_solution(f::Function, o::SCIP.Optimizer, vars::Vector{MOI.Var
     @assert isfinite(best_val)
     return (best_v, best_val)
 end
+
+"""
+Finds the best solution in the HiGHS solution storage, based on the objective function `f`.
+Returns the solution vector and the corresponding best value.
+"""
+function find_best_solution(f::Function, o::HiGHS.Optimizer, vars::Vector{MOI.VariableIndex})
```

Comment:
one thing to note, the ordering of vars is not used here.
Normally, they are supposed to stay 1:n because it's painful to keep track of otherwise but still it should have a check that the variables in HiGHS are the same as the vars you get passed

Reply:
Thanks. I'll implement that